engine/engine.go

```diff
@@ -208,8 +208,6 @@ func (e *EngineAPI) ForkchoiceUpdatedV3(
 		ParentBeaconBlockRoot: pa.ParentBeaconBlockRoot,
 		ParentHash:            fcs.HeadBlockHash,
 		Height:                e.blockStore.HeadBlock().Header.Height + 1,
-		Transactions:          pa.Transactions,
-		CosmosTxs:             cosmosTxs,
```

Comment:
We should keep the `cosmosTxs` field.

Reply:
The problem was fixed by re-running `make setup-e2e'.